<!DOCTYPE HTML>
<html lang="en">
<head>

  <!-- Google tag (gtag.js) -->
  <script async src="https://www.googletagmanager.com/gtag/js?id=G-KG0JZT9Y1R"></script>
  <script>
    window.dataLayer = window.dataLayer || [];
    function gtag(){dataLayer.push(arguments);}
    gtag('js', new Date());

    gtag('config', 'G-KG0JZT9Y1R');
  </script>

  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">

  <title>Ava Amini - Personal Website</title>

  <meta name="author" content="Ava Amini">
  <meta name="description" content="Ava Amini is a Principal Researcher at Microsoft Research, where she develops new AI methods to understand and design biology, with the ultimate aim of realizing precision biomedicines that improve human health. Ava is a lead instructor of MIT Introduction to Deep Learning, and received her PhD from Harvard Biophysics and her Bachelors from MIT.">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="keywords" content="ava amini,ava soleimany,mit deep learning,microsoft,cancer,harvard,machine learning, mit,biomedical,precision medicine" />

  <link rel="stylesheet" type="text/css" href="stylesheet.css">
  <link rel="icon" type="image/png" href="images/mit_logo2.png">

  <link rel="apple-touch-icon" sizes="180x180" href="images/favicon/apple-touch-icon.png">
  <link rel="icon" type="image/png" sizes="32x32" href="images/favicon/favicon-32x32.png">
  <link rel="icon" type="image/png" sizes="16x16" href="images/favicon/favicon-16x16.png">
  <link rel="manifest" href="images/favicon/site.webmanifest">
  <link rel="mask-icon" href="images/favicon/safari-pinned-tab.svg" color="#5bbad5">
  <meta name="msapplication-TileColor" content="#da532c">
  <meta name="theme-color" content="#ffffff">


  <meta property="og:title" content="Ava Amini"/>
  <meta property="og:type" content="article"/>
  <meta property="og:url" content="https://avaamini.com"/>
  <meta property="og:site_name" content="Ava Amini"/>
  <meta name="og:description" content="Ava Amini is a Principal Researcher at Microsoft Research, where she develops new AI methods to understand and design biology, with the ultimate aim of realizing precision biomedicines that improve human health. Ava is a lead instructor of MIT Introduction to Deep Learning, and received her PhD from Harvard Biophysics and her Bachelors from MIT.">
  <meta property="og:image" content="https://avaamini.com/images/Ava_Amini.jpg"/>

</head>

<body>
  <table style="width:100%;max-width:800px;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
    <tr style="padding:0px">
      <td style="padding:0px">
        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
          <tr style="padding:0px">
            <td style="padding:2.5%;width:63%;vertical-align:middle">
              <p style="text-align:center">
                <name>Ava Amini</name>
                <!-- <br> -->
                <!-- <em> Principal Researcher, Microsoft </em> -->
              <!-- </br> -->
              </p>
              <p> I am a Principal Researcher at <a href="https://www.microsoft.com/en-us/research/lab/microsoft-research-new-england/">Microsoft Research in Cambridge, MA</a>.
               
                <br>
                My research focuses on developing new AI methods to understand and design biology, with the ultimate aim of realizing precision biomedicines to improve human health.
                To this end, I co-lead <em><a href="https://exvivo.broadinstitute.org/">Project Ex Vivo</a></em>, a collaborative effort between Microsoft and the Broad Institute, that is focused on defining, engineering, and targeting cell states in cancer.
              </p>
              <p>
                I completed my PhD in Biophysics at <a href="https://biophysics.fas.harvard.edu/">Harvard University</a>, where I worked with <a href="http://lmrt.mit.edu/">Sangeeta Bhatia</a> at the <a href="https://ki.mit.edu/">Koch Institute for Integrative Cancer Research</a>
                and was supported by the <a href="https://www.nsfgrfp.org/">NSF Graduate Research Fellowship</a>.
                I received my Bachelor of Science in Computer Science and Molecular Biology from <a href="https://www.mit.edu/">MIT</a>,
                where I was recognized as a <a href="https://engineering.mit.edu/students/awards/">Henry Ford II Scholar</a>
                and with the <a href="https://amita.alumcommunity.mit.edu/page/amita-academic-award">AMITA Academic Award</a>.
              </p>
              <p style="text-align:center">
                <a href="#research">Research</a> &nbsp/&nbsp
                <a href="#teaching">Teaching</a> &nbsp/&nbsp
                <a href="#talks">Talks</a>
              </p>
              <p style="text-align:center">
                <a href="mailto:ava.amini@microsoft.com">Email</a> &nbsp/&nbsp
                <a href="data/Ava_Amini_CV.pdf">CV</a> &nbsp/&nbsp
                <a href="https://scholar.google.com/citations?user=w_wosd4AAAAJ&hl=en">Google Scholar</a> &nbsp/&nbsp
                <a href="https://twitter.com/avapamini">Twitter</a>
              </p>
            </td>
            <td style="padding:2.5%;width:40%;max-width:40%">
              <a href="images/Ava_Amini.jpg"><img style="width:100%;max-width:100%" alt="profile photo" src="images/Ava_Amini.jpg" class="hoverZoomLink"></a>
            </td>
          </tr>
        </tbody></table>

        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>



            <tr>
            <td style="padding:20px;width:100%;vertical-align:middle">
              <heading id="research">Research and Publications</heading>
              <p>
              My research focuses on developing new AI methods to understand and design biology, with the ultimate aim of realizing precision biomedicines to improve human health. My work bridges the computational and experimental worlds to create new diagnostic and therapeutic biotechnologies and to achieve new insights into cancer.
              </p>
              <strong>Formerly Ava Soleimany. *Denotes co-first authorship. <sup>&#8224;</sup>Denotes corresponding authors</strong>.
            </td>
          </tr>
        </tbody></table>
        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>

          <tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <div class="one">
              <img src='images/papers/p_2025_hce.jpg'>
              </div>
            </td>
            <td style="padding:20px;width:75%;vertical-align:middle">
              <a href="https://www.biorxiv.org/content/10.1101/2025.04.23.650210v1">
                <papertitle>Hierarchical cross-entropy loss improves atlas-scale single-cell annotation models</papertitle>
              </a>
              <br>
              Sebastiano Cultrera di Montesano,
              Davide D'Ascenzo,
              Srivatsan Raghavan,
              <strong>Ava P. Amini</strong>,
              Peter S. Winter,
              Lorin Crawford
              <br>
              <em>bioRxiv</em>, 2025
              <br>
              <a href="https://www.biorxiv.org/content/10.1101/2025.04.23.650210v1.full.pdf">pdf</a> /
              <a href="https://github.com/microsoft/hce-classification">code</a>
              <br>
              <p> We introduce a hierarchical cross-entropy loss for single-cell cell type annotation, finding that this simple modification significantly improves out-of-distribution performance without added computational cost.</p>
            </td>
          </tr>



            <tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <div class="one">
              <img src='images/papers/p_2025_zero-shot.jpg'>
              </div>
            </td>
            <td style="padding:20px;width:75%;vertical-align:middle">
              <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-025-03574-x">
                <papertitle>Zero-shot evaluation reveals limitations of single-cell foundation models</papertitle>
              </a>
              <br>
              Kasia Z. Kedzierska,
              Lorin Crawford,
              <strong>Ava P. Amini</strong>,
              Alex X. Lu
              <br>
              <em>Genome Biology</em>, 2025
              <br>
              <a href="https://genomebiology.biomedcentral.com/counter/pdf/10.1186/s13059-025-03574-x.pdf">pdf</a> /
              <a href="https://github.com/microsoft/zero-shot-scfoundation">code</a>
              <br>
              <p> We evaluate the performance of single-cell foundation models in ``zero-shot`` settings where they are used without any further training, finding that they are outperformed by simpler methods.</p>
            </td>
          </tr>


          <tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <div class="one">
              <img src='images/papers/p_2025_protnote.jpg'>
              </div>
            </td>
            <td style="padding:20px;width:75%;vertical-align:middle">
              <a href="https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btaf170/8113843">
                <papertitle>ProtNote: a multimodal method for protein-function annotation</papertitle>
              </a>
              <br>
              Samir Char,
              Nathaniel Corley,
              Sarah Alamdari,
              Kevin K. Yang,
              <strong>Ava P. Amini<sup>&#8224;</sup></strong>
              <br>
              <em>Bioinformatics</em>, 2025
              <br>
              <a href="https://github.com/microsoft/protnote">code</a>
              <br>
              <p> ProtNote is a multimodal deep learning model that leverages free-form text to enable both supervised and zero-shot protein function prediction.</p>
            </td>
          </tr>
        

          <tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <div class="one">
              <img src='images/papers/p_2025_micon.jpg'>
              </div>
            </td>
            <td style="padding:20px;width:75%;vertical-align:middle">
              <a href="https://arxiv.org/abs/2504.09544">
                <papertitle>Causal integration of chemical structures improves representations of microscopy images for morphological profiling</papertitle>
              </a>
              <br>
              Yemin Yu,
              Neil Tenenholtz,
              Lester Mackey,
              Ying Wei,
              David Alvarez-Melis,
              <strong>Ava P. Amini</strong>,
              Alex X. Lu
              <br>
              <em>arXiv</em>, 2025
              <br>
              <a href="https://arxiv.org/pdf/2504.09544">pdf</a> /
              <a href="https://github.com/microsoft/micon">code</a>
              <br>
              <p> We introduce a new deep learning framework, MICON, that models chemical compounds as treatments that induce counterfactual transformations of cell phenotypes for improved representation learning in microscopy-based morphological profiling.</p>
            </td>
          </tr>

          <tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <div class="one">
              <img src='images/papers/p_2025_recall.jpg'>
              </div>
            </td>
            <td style="padding:20px;width:75%;vertical-align:middle">
              <a href="https://www.cell.com/ajhg/abstract/S0002-9297(25)00061-8">
                <papertitle>Artificial variables help to avoid over-clustering in single-cell RNA sequencing</papertitle>
              </a>
              <br>
              Alan DenAdel,
              Michelle L. Ramseier,
              Andrew W. Navia,
              Alex K. Shalek,
              Srivatsan Raghavan,
              Peter S. Winter,
              <strong>Ava P. Amini</strong>,
              Lorin Crawford
              <br>
              <em>American Journal of Human Genetics</em>, 2025
              <br>
              <a href="https://github.com/lcrawlab/recall">code</a>
              <br>
              <p> We develop a method for protecting against over-clustering in analysis of single-cell RNA sequencing data, by controlling for the impact of reusing the same data twice when performing differential expression analysis.</p>
            </td>
          </tr>

          <tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <div class="one">
              <img src='images/papers/p_2025_cleavenet.jpg'>
              </div>
            </td>
            <td style="padding:20px;width:75%;vertical-align:middle">
              <a href="https://www.biorxiv.org/content/10.1101/2025.02.27.640681.abstract">
                <papertitle>Deep learning guided design of protease substrates</papertitle>
              </a>
              <br>
              Carmen Martin-Alonso*,
              Sarah Alamdari*,
              Tahoura S. Samad,
              Kevin K. Yang,
              Sangeeta N. Bhatia<sup>&#8224;</sup>,
              <strong>Ava P. Amini<sup>&#8224;</sup></strong>
              <br>
              <em>bioRxiv</em>, 2025
              <br>
              <a href="https://www.biorxiv.org/content/biorxiv/early/2025/03/02/2025.02.27.640681.full.pdf">pdf</a> /
              <a href="https://github.com/microsoft/cleavenet">code</a>
              <br>
              <p> We present CleaveNet, an end-to-end AI pipeline for the design of peptide-based protease substrates, enabling generation of peptides guided by a target cleavage profile for the design of efficient and selective substrates.</p>
            </td>
          </tr>
        

          <tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <div class="one">
              <img src='images/papers/p_2025_cogen.jpg'>
              </div>
            </td>
            <td style="padding:20px;width:75%;vertical-align:middle">
              <a href="https://www.sciencedirect.com/science/article/abs/pii/S0959440X25000363">
                <papertitle>Toward deep learning sequenceâ€“structure co-generation for protein design</papertitle>
              </a>
              <br>
              Chentong Wang,
              Sarah Alamdari,
              Carles Domingo-Enrich,
              <strong>Ava P. Amini</strong>,
              Kevin K. Yang
              <br>
              <em>Current Opinion in Structural Biology</em>, 2025
              <br>
              <p> We review recent advances in deep generative models for protein design, with a particular focus on sequence-structure co-generation methods.</p>
            </td>
          </tr>


            <tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <div class="one">
              <img src='images/papers/p_2025_mix.jpg'>
              </div>
            </td>
            <td style="padding:20px;width:75%;vertical-align:middle">
              <a href="https://www.biorxiv.org/content/10.1101/2025.02.19.639127.abstract">
                <papertitle>Consequences of training data composition for deep learning models in single-cell biology</papertitle>
              </a>
              <br>
              Ajay Nadig<sup>&#8224;</sup>,
              Akshaya Thoutam,
              Madeline Hughes,
              Anay Gupta,
              Andrew W. Navia,
              Nicolo Fusi,
              Srivatsan Raghavan,
              Peter S. Winter,
              <strong>Ava P. Amini<sup>&#8224;</sup></strong>,
              Lorin Crawford<sup>&#8224;</sup>
              <br>
              <em>bioRxiv</em>, 2025
              <br>
              <a href="https://www.biorxiv.org/content/10.1101/2025.02.19.639127.full.pdf">pdf</a> /
              <a href="https://github.com/microsoft/scFM-datamix">code</a>
              <br>
              <p> We systematically investigate the consequences of training dataset composition on the behavior of deep learning models of single-cell transcriptomics, focusing on human hematopoiesis as a tractable model system and including cells from adult and developing tissues, disease states, and perturbation atlases.</p>
            </td>
          </tr>
          
          <tr onmouseout="p_2023_proteinUQ_stop()" onmouseover="p_2023_proteinUQ_start()">
            <td style="padding:20px;width:25%;vertical-align:middle">
              <div class="one">
                <div class="two" id='p_2023_proteinUQ_image'><img src='images/papers/p_2023_proteinUQ_after.jpg'></div>
                <img src='images/papers/p_2023_proteinUQ_before.jpg'>
              </div>
              <script type="text/javascript">
                function p_2023_proteinUQ_start() {
                  document.getElementById('p_2023_proteinUQ_image').style.opacity = "1";
                }

                function p_2023_proteinUQ_stop() {
                  document.getElementById('p_2023_proteinUQ_image').style.opacity = "0";
                }
                p_2023_proteinUQ_stop()
              </script>
            </td>
            <td style="padding:20px;width:75%;vertical-align:middle">
              <a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1012639">
                <papertitle>Benchmarking uncertainty quantification for protein engineering</papertitle>
              </a>
              <br>
              Kevin P. Greenman,
              <strong>Ava P. Amini<sup>&#8224;</sup></strong>,
              Kevin K. Yang<sup>&#8224;</sup>
              <br>
              <em>PLOS Computational Biology</em>, 2025
              <br>
              <a href="https://journals.plos.org/ploscompbiol/article/file?id=10.1371/journal.pcbi.1012639&type=printable">pdf</a> /
              <a href="https://github.com/microsoft/protein-uq">code</a>
              <br>
              <p> We assess deep learning-based uncertainty quantification methods
                on protein sequence-function prediction tasks.</p>
            </td>
          </tr>

          <tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <div class="one">
              <img src='images/papers/p_2024_scSFM.jpg'>
              </div>
            </td>
            <td style="padding:20px;width:75%;vertical-align:middle">
              <a href="https://www.biorxiv.org/content/10.1101/2024.12.13.628448.abstract">
                <papertitle>Evaluating the role of pre-training dataset size and diversity on single-cell foundation model performance</papertitle>
              </a>
              <br>
              Alan DenAdel,
              Madeline Hughes,
              Akshaya Thoutam,
              Anay Gupta,
              Andrew W. Navia,
              Nicolo Fusi,
              Srivatsan Raghavan,
              Peter S. Winter,
              <strong>Ava P. Amini<sup>&#8224;</sup></strong>,
              Lorin Crawford<sup>&#8224;</sup>
              <br>
              <em>bioRxiv</em>, 2024
              <br>
              <a href="https://www.biorxiv.org/content/10.1101/2024.12.13.628448.full.pdf">pdf</a> /
              <a href="https://github.com/microsoft/scFM-dataselection">code</a>
              <br>
              <p> We investigate the role of pre-training dataset size and diversity on the performance of single-cell foundation models on both zero-shot and fine-tuned tasks, finding that current methods plateau in performance with pre-training datasets that are only a fraction of the full size. </p>
            </td>
          </tr>

          
          <tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <div class="one">
              <img src='images/papers/p_2024_scbert.jpg'>
              </div>
            </td>
            <td style="padding:20px;width:75%;vertical-align:middle">
              <a href="https://www.nature.com/articles/s42256-024-00949-w">
                <papertitle>Deeper evaluation of a single-cell foundation model</papertitle>
              </a>
              <br>
              Rebecca Boiarsky,
              Nalini M. Singh,
              Alejandro Buendia,
              <strong>Ava P. Amini</strong>,
              Gad Getz,
              David Sontag
              <br>
              <em>Nature Machine Intelligence</em>, 2024
              <br>
              <a href="https://www.nature.com/articles/s42256-024-00949-w.pdf">pdf</a> /
              <a href="https://github.com/clinicalml/sc-foundation-eval/tree/v1.0">code</a>
              <br>
              <p> We take a deep dive into scBERT, one recently developed transformer model for single-cell RNA-sequencing data, to develop a deeper understanding of the potential benefits and limitations of single-cell foundation models.</p>
            </td>
          </tr>

          <tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <div class="one">
              <img src='images/papers/p_2024_evodiff.jpg'>
              </div>
            </td>
            <td style="padding:20px;width:75%;vertical-align:middle">
              <a href="https://www.biorxiv.org/content/10.1101/2023.09.11.556673v2.abstract">
                <papertitle>Protein generation with evolutionary diffusion: sequence is all you need</papertitle>
              </a>
              <br>
              Sarah Alamdari,
              Nitya Thakkar,
              Rianne van den Berg,
              Neil Tenenholtz,
              Robert Strome,
              Alan M. Moses,
              Alex X. Lu,
              Nicolo Fusi,
              <strong>Ava P. Amini<sup>&#8224;</sup></strong>,
              Kevin K. Yang<sup>&#8224;</sup>
              <br>
              <em>bioRxiv</em>, 2024
              <br>
              <a href="https://www.biorxiv.org/content/10.1101/2023.09.11.556673v2.full.pdf">pdf</a> /
              <a href="https://github.com/microsoft/evodiff">code</a>
              <br>
              <p> We develop EvoDiff, a general-purpose discrete diffusion over protein sequences, that combines evolutionary-scale data with the distinct conditioning capabilities of diffusion models for controllable protein generation in sequence space.</p>
            </td>
          </tr>

          
          <tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <div class="one">
              <img src='images/papers/p_2024_compressed.jpg'>
              </div>
            </td>
            <td style="padding:20px;width:75%;vertical-align:middle">
              <a href="https://www.nature.com/articles/s41587-024-02403-z">
                <papertitle>Scalable, compressed phenotypic screening using pooled perturbations</papertitle>
              </a>
              <br>
              Nuo Liu,
              Walaa E. Kattan,
              Benjamin E. Mead,
              Conner Kummerlowe,
              Thomas Cheng,
              Sarah Ingabire,
              Jamie H. Cheah,
              Christian K. Soule,
              Anita Vrcic,
              Jane K. McIninch,
              Sergio Triana,
              Manuel Guzman,
              Tyler T. Dao,
              Joshua M. Peters,
              Kristen E. Lowder,
              Lorin Crawford,
              <strong>Ava P. Amini</strong>,
              Paul C. Blainey,
              William C. Hahn,
              Brian Cleary,
              Bryan Bryson,
              Peter S. Winter,
              Srivatsan Raghavan,
              Alex K. Shalek
              <br>
              <em>Nature Biotechnology</em>, 2024
              <br>
              <a href="https://www.nature.com/articles/s41587-024-02403-z.pdf">pdf</a>
              <br>
              <p> We establish a method of pooling perturbations, like chemical compounds, followed by computational deconvolution to reduce required sample size, labor, and cost in high-throughput phenotypic screens.</p>
            </td>
          </tr>


          <tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <div class="one">
              <img src='images/papers/p_2024_all.jpg'>
              </div>
            </td>
            <td style="padding:20px;width:75%;vertical-align:middle">
              <a href="https://www.biorxiv.org/content/10.1101/2024.06.06.597767v1">
                <papertitle>Mutation and cell state compatibility is required and targetable in Ph+ acute lymphoblastic leukemia minimal residual disease</papertitle>
              </a>
              <br>
              Peter S. Winter*,
              Michelle L. Ramseier*,
              Andrew W. Navia*,
              Sachit Saksena,
              Haley Strouf,
              Nezha Senhaji,
              Alan DenAdel,
              Mahnoor Mirza,
              Hyun Hwan An,
              Laura Bilal,
              Peter Dennis,
              Catherine S. Leahy,
              Kay Shigemori,
              Jennyfer Galves-Reyes,
              Ye Zhang,
              Foster Powers,
              Nolawit Mulugeta,
              Alejandro J. Gupta,
              Nicholas Calistri,
              Alex Van Scoyk, Kristen Jones,
              Huiyun Liu,
              Kristen E. Stevenson,
              Siyang Ren,
              Marlise R. Luskin,
              Charles P. Couturier,
              <strong>Ava P. Amini</strong>,
              Srivatsan Raghavan,
              Robert J. Kimmerling,
              Mark M. Stevens,
              Lorin Crawford,
              David M. Weinstock,
              Scott R. Manalis,
              Alex K. Shalek,
              Mark A. Murakami
              <br>
              <em>bioRxiv</em>, 2024
              <br>
              <a href="https://www.biorxiv.org/content/10.1101/2024.06.06.597767v1.full.pdf">pdf</a>
              <br>
              <p> Utilizing patient-derived xenograft (PDX) models and clinical trial specimens of acute lymphoblastic leukemia (ALL), we examined how genetic and transcriptional features co-evolve to drive progression during prolonged tyrosine kinase inhibitor response, uncovering a landscape of cooperative mutational and transcriptional escape mechanisms that differ from those causing resistance to first generation inhibitors.</p>
            </td>
          </tr>

          <tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <div class="one">
              <img src='images/papers/p_2024_transfer.jpg'>
              </div>
            </td>
            <td style="padding:20px;width:75%;vertical-align:middle">
              <a href="https://proceedings.mlr.press/v235/li24a.html">
                <papertitle>Feature reuse and scaling: Understanding transfer learning with protein language models</papertitle>
              </a>
              <br>
              Francesca-Zhoufan Li,
              <strong>Ava P. Amini</strong>,
              Yisong Yue,
              Kevin K. Yang,
              Alex X. Lu
              <br>
              <em>International Conference on Machine Learning</em>, 2024
              <br>
              <a href="https://raw.githubusercontent.com/mlresearch/v235/main/assets/li24a/li24a.pdf">pdf</a>
              <br>
              <p> To understand how the features learned in pretraining protein language models (PLMs) relate to and are useful for downstream tasks, we perform a systematic analysis of transfer learning using PLMs, conducting 370 experiments across a comprehensive suite of factors including different downstream tasks, architectures, model sizes, model depths, and pretraining time.</p>
            </td>
          </tr>

          <tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <div class="one">
              <img src='images/papers/p_2024_nclusion.jpg'>
              </div>
            </td>
            <td style="padding:20px;width:75%;vertical-align:middle">
              <a href="https://www.biorxiv.org/content/10.1101/2024.02.11.579839v1">
                <papertitle>Scalable nonparametric clustering with unified marker gene selection for single-cell RNA-seq data</papertitle>
              </a>
              <br>
              Chibuikem Nwizu,
              Madeline Hughes,
              Michelle L. Ramseier,
              Andrew W. Navia,
              Alex K. Shalek,
              Nicolo Fusi,
              Srivatsan Raghavan,
              Peter S. Winter,
              <strong>Ava P. Amini<sup>&#8224;</sup></strong>,
              Lorin Crawford<sup>&#8224;</sup>
              <br>
              <em>bioRxiv</em>, 2024
              <br>
              <a href="https://www.biorxiv.org/content/10.1101/2024.02.11.579839v1.full.pdf">pdf</a>
              <br>
              <p> We developed a non-parametric infinite mixture model that leverages Bayesian sparse priors to identify marker genes while simultaneously performing clustering on single-cell expression data.</p>
            </td>
          </tr>

          <tr onmouseout="p_2023_foldingdiff_stop()" onmouseover="p_2023_foldingdiff_start()">
            <td style="padding:20px;width:25%;vertical-align:middle">
              <div class="one">
                <img src='images/papers/p_2023_foldingdiff_anim.gif'>
              </div>
            </td>
            <td style="padding:20px;width:75%;vertical-align:middle">
              <a href="https://www.nature.com/articles/s41467-024-45051-2">
                <papertitle>Protein structure generation via folding diffusion</papertitle>
              </a>
              <br>
              Kevin E. Wu,
              Kevin K. Yang,
              Rianne van den Berg,
              Sarah Alamdari,
              James Y. Zou,
              Alex X. Lu,
              <strong>Ava P. Amini<sup>&#8224;</sup></strong>
              <br>
              <em>Nature Communications</em>, 2024
              <br>
              <a href="https://www.nature.com/articles/s41467-024-45051-2.pdf">pdf</a> / 
              <a href="https://github.com/microsoft/foldingdiff">code</a>
              <br>
              <p> We present FoldingDiff, a diffusion-based generative model that generates protein backbone structures via a procedure inspired by the natural folding process.</p>
            </td>
          </tr>

          <tr onmouseout="p_2023_ctDNA_stop()" onmouseover="p_2023_ctDNA_start()">
            <td style="padding:20px;width:25%;vertical-align:middle">
              <div class="one">
                <div class="two" id='p_2023_ctDNA_image'><img src='images/papers/p_2023_ctDNA_after.jpg'></div>
                <img src='images/papers/p_2023_ctDNA_before.jpg'>
              </div>
              <script type="text/javascript">
                function p_2023_ctDNA_start() {
                  document.getElementById('p_2023_ctDNA_image').style.opacity = "1";
                }

                function p_2023_ctDNA_stop() {
                  document.getElementById('p_2023_ctDNA_image').style.opacity = "0";
                }
                p_2022_Ionic_stop()
              </script>
            </td>
            <td style="padding:20px;width:75%;vertical-align:middle">
              <a href="https://www.science.org/doi/abs/10.1126/science.adf2341">
                <papertitle>Priming agents transiently reduce the clearance of cell-free DNA to improve liquid biopsies</papertitle>
              </a>
              <br>
              Carmen Martin-Alonso*,
              Shervin Tabrizi*,
              Kan Xiong*,
              Timothy Blewett,
              Sainetra Sridhar,
              Andjela Crnjac,
              Sahil Patel,
              Zhenyi An,
              Ahmet Bekdemir,
              Douglas Shea,
              Shih-Ting Wang,
              Sergio Rodriguez-Aponte,
              Christopher A. Naranjo,
              Justin Rhoades,
              Jesse D. Kirkpatrick,
              Heather E. Fleming,
              <strong>Ava P. Amini</strong>,
              Todd R. Golub,
              J. Christopher Love<sup>&#8224;</sup>,
              Sangeeta N. Bhatia<sup>&#8224;</sup>,
              Viktor A. Adalsteinsson<sup>&#8224;</sup>
              <br>
              <em>Science</em>, 2024
              <br>
              <a href="https://www.lmrt.mit.edu/s/Martin-Alonso_Science_2024.pdf">pdf</a> /
              <a href="https://news.mit.edu/2024/researchers-improve-blood-tests-ability-detect-monitor-cancer-0118"> MIT press</a> /
              <a href="https://phys.org/news/2024-01-agents-liquid-biopsy-cancer.html"> general press</a>
              <br>
              <p> We develop intravenous priming agents that are given prior to a blood draw to increase the
                abundance of cell-free DNA in circulation, improving the sensitivity of liquid biopsy cancer diagnostic assays.</p>
            </td>
          </tr>

                    <tr onmouseout="p_2022_BGC_stop()" onmouseover="p_2022_BGC_start()">
            <td style="padding:20px;width:25%;vertical-align:middle">
              <div class="one">
                <div class="two" id='p_2022_BGC_image'><img src='images/papers/p_2022_BGC_after.jpg'></div>
                <img src='images/papers/p_2022_BGC_before.jpg'>
              </div>
              <script type="text/javascript">
                function p_2022_BGC_start() {
                  document.getElementById('p_2022_BGC_image').style.opacity = "1";
                }

                function p_2022_BGC_stop() {
                  document.getElementById('p_2022_BGC_image').style.opacity = "0";
                }
                p_2022_BGC_stop()
              </script>
            </td>
            <td style="padding:20px;width:75%;vertical-align:middle">
              <a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1011162">
                <papertitle>Deep self-supervised learning for biosynthetic gene
                  cluster detection and product classification</papertitle>
              </a>
              <br>
              Carolina Rios-Martinez,
              Nicholas Bhattacharya,
              <strong>Ava P. Amini</strong>,
              Lorin Crawford,
              Kevin K. Yang
              <br>
              <em>PLOS Computational Biology</em>, 2023
              <br>
              <a href="https://journals.plos.org/ploscompbiol/article/file?id=10.1371/journal.pcbi.1011162&type=printable">pdf</a>
              <br>
              <p> We develop a self-supervised masked language model for biosynthetic gene
                clusters in bacteria, and leverage it for natural product classification.</p>
            </td>
          </tr>

          <tr onmouseout="p_2022_BGC_stop()" onmouseover="p_2022_BGC_start()">
            <td style="padding:20px;width:25%;vertical-align:middle">
              <div class="one">
              <img src='images/papers/p_2023_diffusion.jpg'>
              </div>
            </td>
            <td style="padding:20px;width:75%;vertical-align:middle">
              <a href="https://www.nature.com/articles/s43588-023-00454-x">
                <papertitle>From noise to protein with image models</papertitle>
              </a>
              <br>
              <strong>Ava P. Amini</strong>,
              Kevin K. Yang
              <br>
              <em>Nature Computational Science</em>, 2023
              <br>
              <a href="https://www.nature.com/articles/s43588-023-00454-x.pdf">pdf</a>
              <br>
              <p> A commentary on a recent image-inspired, diffusion model to generate new protein structures.</p>
            </td>
          </tr>

          <tr onmouseout="p_2023_LN_stop()" onmouseover="p_2023_LN_start()">
            <td style="padding:20px;width:25%;vertical-align:middle">
              <div class="one">
                <div class="two" id='p_2023_LN_image'><img src='images/papers/p_2023_LN_after.jpg'></div>
                <img src='images/papers/p_2023_LN_before.jpg'>
              </div>
              <script type="text/javascript">
                function p_2023_LN_start() {
                  document.getElementById('p_2023_LN_image').style.opacity = "1";
                }

                function p_2023_LN_stop() {
                  document.getElementById('p_2023_LN_image').style.opacity = "0";
                }
                p_2022_Ionic_stop()
              </script>
            </td>
            <td style="padding:20px;width:75%;vertical-align:middle">
              <a href="https://www.science.org/doi/abs/10.1126/science.abn8934">
                <papertitle>Low protease activity in B cell follicles promotes retention of intact antigens after immunization</papertitle>
              </a>
              <br>
              Aereas Aung,
              Ang Cui,
              Laura Maiorino,
              <strong>Ava P. Amini</strong>,
              Justin R. Gregory,
              Maurice Bukenya,
              Yiming Zhang,
              Heya Lee,
              Christopher A. Cottrell,
              Duncan M. Morgan,
              Murillo Silva,
              Heikyung Suh,
              Jesse D. Kirkpatrick,
              Parastoo Amlashi,
              Tanaka Remba,
              Leah M. Froehle,
              Shuhao Xiao,
              Wuhbet Abraham,
              Josetta Adams,
              J. Christopher Love,
              Phillip Huyett,
              Douglas S. Kwon,
              Nir Hacohen,
              William R. Schief,
              Sangeeta N. Bhatia,
              Darrell J. Irvine
              <br>
              <em>Science</em>, 2023
              <br>
              <a href="https://www.nature.com/articles/s41467-022-32988-5.pdf">pdf</a> /
              <a href="https://ki.mit.edu/news/2023/safe-haven-for-vaccine-antigens"> MIT press</a> /
              <a href="https://www.the-scientist.com/news-opinion/targeting-antigen-sanctuary-in-lymph-nodes-could-make-vaccines-better-70943"> general press</a>
              <br>
              <p> We discover "sanctuaries" within lymph nodes that contain low proteolytic activity and
                act as a safe haven for vaccines, and demonstrate that this heterogeneity
                can be exploited to enhance vaccine-induced antibody response.</p>
            </td>
          </tr>


<!-- 
          <tr onmouseout="p_2023_foldingdiff_stop()" onmouseover="p_2023_foldingdiff_start()">
            <td style="padding:20px;width:25%;vertical-align:middle">
              <div class="one">
                <img src='images/papers/p_2023_foldingdiff_anim.gif'>
              </div>
            </td>
            <td style="padding:20px;width:75%;vertical-align:middle">
              <a href="https://arxiv.org/abs/2209.15611">
                <papertitle>Protein structure generation via folding diffusion</papertitle>
              </a>
              <br>
              Kevin E. Wu,
              Kevin K. Yang,
              Rianne van den Berg,
              James Y. Zou,
              Alex X. Lu,
              <strong>Ava P. Amini<sup>&#8224;</sup></strong>

              <br>
              <em>arXiv</em>, 2022
              <br>
              <a href="https://arxiv.org/pdf/2209.15611.pdf">pdf</a> /
              <a href="https://github.com/microsoft/foldingdiff">code</a> /
              <a href="https://huggingface.co/spaces/wukevin/foldingdiff">huggingface</a>
              <br>
              <p> We present Folding Diffusion: a diffusion model for generating protein structures
                that is inspired by the biophysics of protein folding.</p>
            </td>
          </tr> -->

          <tr onmouseout="p_2021_EA_stop()" onmouseover="p_2021_EA_start()">
            <td style="padding:20px;width:25%;vertical-align:middle">
              <div class="one">
                <div class="two" id='p_2021_EA_image'><img src='images/papers/p_2021_EA_after.jpg'></div>
                <img src='images/papers/p_2021_EA_before.jpg'>
              </div>
              <script type="text/javascript">
                function p_2021_EA_start() {
                  document.getElementById('p_2021_EA_image').style.opacity = "1";
                }

                function p_2021_EA_stop() {
                  document.getElementById('p_2021_EA_image').style.opacity = "0";
                }
                p_2021_EA_stop()
              </script>
            </td>
            <td style="padding:20px;width:75%;vertical-align:middle">
              <a href="https://www.nature.com/articles/s41467-022-32988-5">
                <papertitle>Multiscale profiling of protease activity in cancer</papertitle>
              </a>
              <br>
              <strong>Ava P. Amini*</strong>,
              Jesse D. Kirkpatrick*,
              Cathy S. Wang,
              Alex M. Jaeger,
              Susan Su,
              Santiago Naranjo,
              Qian Zhong,
              Tyler Jacks,
              Sangeeta N. Bhatia
              <br>
              <em>Nature Communications</em>, 2022
              <br>
              <a href="https://www.nature.com/articles/s41467-022-32988-5.pdf">pdf</a> /
              <a href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41467-022-32988-5/MediaObjects/41467_2022_32988_MOESM1_ESM.pdf">supplement</a>
              <br>
              <p> We engineer an integrated set of methods for measuring specific protease activities
                across the organismal, tissue, and cellular scales, and unify these methods into a
                methodological hierarchy that powers new biological insights into cancer.</p>
            </td>
          </tr>

          <tr onmouseout="p_2022_PAA_stop()" onmouseover="p_2022_PAA_start()">
            <td style="padding:20px;width:25%;vertical-align:middle">
              <div class="one">
                <div class="two" id='p_2022_PAA_image'><img src='images/papers/p_2022_PAA_after.jpg'></div>
                <img src='images/papers/p_2022_PAA_before.jpg'>
              </div>
              <script type="text/javascript">
                function p_2022_PAA_start() {
                  document.getElementById('p_2022_PAA_image').style.opacity = "1";
                }

                function p_2022_BV_stop() {
                  document.getElementById('p_2022_PAA_image').style.opacity = "0";
                }
                p_2022_PAA_stop()
              </script>
            </td>
            <td style="padding:20px;width:75%;vertical-align:middle">
              <a href="https://pubs.acs.org/doi/full/10.1021/acsomega.2c01559">
                <papertitle>Protease Activity Analysis: A toolkit for analyzing
                enzyme activity data</papertitle>
              </a>
              <br>
              <strong>Ava P. Soleimany*</strong>,
              Carmen Martin-Alonso*,
              Melodi Anahtar*,
              Cathy S. Wang,
              Sangeeta N. Bhatia
              <br>
              <em>ACS Omega</em>, 2022
              <br>
              <a href="https://pubs.acs.org/doi/pdf/10.1021/acsomega.2c01559">pdf</a> /
              <a href="https://github.com/avaamini/protease_activity_analysis">code</a>
              <br>
              <p> We build Protease Activity Analysis (PAA), a Python software
                package with a collection of data analytic and machine learning
                tools for analyzing protease activity data.</p>
            </td>
          </tr>

          <tr onmouseout="p_2022_BV_stop()" onmouseover="p_2022_BV_start()">
            <td style="padding:20px;width:25%;vertical-align:middle">
              <div class="one">
                <div class="two" id='p_2022_BV_image'><img src='images/papers/p_2022_BV_after.jpg'></div>
                <img src='images/papers/p_2022_BV_before.jpg'>
              </div>
              <script type="text/javascript">
                function p_2022_BV_start() {
                  document.getElementById('p_2022_BV_image').style.opacity = "1";
                }

                function p_2022_BV_stop() {
                  document.getElementById('p_2022_BV_image').style.opacity = "0";
                }
                p_2022_BV_stop()
              </script>
            </td>
            <td style="padding:20px;width:75%;vertical-align:middle">
              <a href="https://www.pnas.org/doi/10.1073/pnas.2121778119">
                <papertitle>Host protease activity classifies pneumonia etiology</papertitle>
              </a>
              <br>
              Melodi Anahtar,
              Leslie W. Chan,
              Henry Ko,
              Aditya Rao,
              <strong>Ava P. Soleimany</strong>,
              Purvesh Khatri,
              Sangeeta N. Bhatia
              <br>
              <em>PNAS</em>, 2022
              <br>
              <a href="https://www.pnas.org/doi/epdf/10.1073/pnas.2121778119">pdf</a> /
              <a href="https://github.com/avaamini/protease_activity_analysis">code</a> /
              <a href="https://news.mit.edu/2022/sensor-viral-bacterial-pneumonia-0613">press</a>
              <br>
              <p> We develop a sensor-based, ML-driven system to diagnose pneumonia and
                classify its etiology, using machine learning to classify directly from
                molecular barcodes.</p>
            </td>
          </tr>

          <tr onmouseout="p_2021_LAM_stop()" onmouseover="p_2021_LAM_start()">
            <td style="padding:20px;width:25%;vertical-align:middle">
              <div class="one">
                <div class="two" id='p_2021_LAM_image'><img src='images/papers/p_2021_LAM_after.jpg'></div>
                <img src='images/papers/p_2021_LAM_before.jpg'>
              </div>
              <script type="text/javascript">
                function p_2021_LAM_start() {
                  document.getElementById('p_2021_LAM_image').style.opacity = "1";
                }

                function p_2021_LAM_stop() {
                  document.getElementById('p_2021_LAM_image').style.opacity = "0";
                }
                p_2021_LAM_stop()
              </script>
            </td>
            <td style="padding:20px;width:75%;vertical-align:middle">
              <a href="https://erj.ersjournals.com/content/early/2021/08/19/13993003.00664-2021">
                <papertitle>Protease activity sensors enable real-time treatment response monitoring in lymphangioleiomyomatosis</papertitle>
              </a>
              <br>
              Jesse D. Kirkpatrick,
              <strong>Ava P. Soleimany</strong>,
              Jaideep S. Dudani,
              Heng-Jia Liu,
              Hilaire C. Lam,
              Carmen Priolo,
              Elizabeth P. Henske,
              Sangeeta N. Bhatia
              <br>
              <em>European Respiratory Journal</em>, 2022
              <br>
              <a href="https://erj.ersjournals.com/content/erj/early/2021/08/19/13993003.00664-2021.full.pdf">pdf</a>
              <br>
              <p> We establish a sensor-based, ML-driven diagnostic for noninvasive, real-time monitoring of disease in a preclinical model
                of lymphangioleiomyomatosis (LAM), a rare lung disease.</p>
            </td>
          </tr>



          <tr onmouseout="p_2022_Ionic_stop()" onmouseover="p_2022_Ionic_start()">
            <td style="padding:20px;width:25%;vertical-align:middle">
              <div class="one">
                <div class="two" id='p_2022_Ionic_image'><img src='images/papers/p_2022_ionic_after.jpg'></div>
                <img src='images/papers/p_2022_ionic_before.jpg'>
              </div>
              <script type="text/javascript">
                function p_2022_Ionic_start() {
                  document.getElementById('p_2022_Ionic_image').style.opacity = "1";
                }

                function p_2022_Ionic_stop() {
                  document.getElementById('p_2022_Ionic_image').style.opacity = "0";
                }
                p_2022_Ionic_stop()
              </script>
            </td>
            <td style="padding:20px;width:75%;vertical-align:middle">
              <a href="https://onlinelibrary.wiley.com/doi/10.1002/adhm.202102685">
                <papertitle>Ionic liquid-mediated transdermal delivery of
                thrombosis-detecting nanosensors</papertitle>
              </a>
              <br>
              Ahmet Bekdemir,
              Eden E.L. Tanner,
              Jesse D. Kirkpatrick,
              <strong>Ava P. Soleimany</strong>,
              Samir Mitragotri,
              Sangeeta N. Bhatia
              <br>
              <em>Advanced Healthcare Materials</em>, 2022
              <br>
              <a href="https://onlinelibrary.wiley.com/doi/epdf/10.1002/adhm.202102685">pdf</a> /
              <a href="https://onlinelibrary.wiley.com/action/downloadSupplement?doi=10.1002%2Fadhm.202102685&file=adhm202102685-sup-0001-SuppMat.pdf">supplement</a>
              <br>
              <p> We design an easily applicable, non-invasive formulation to deliver diagnostic nanosensors through
                the skin, enabling a sustained release diagnostic monitoring system for detecting thrombosis.</p>
            </td>
          </tr>


<!--
OLD PUBLICATIONS BELOW -->

          <tr onmouseout="p_2021_TEV_stop()" onmouseover="p_2021_TEV_start()">
            <td style="padding:20px;width:25%;vertical-align:middle">
              <div class="one">
                <div class="two" id='p_2021_TEV_image'><img src='images/papers/p_2021_TEV_after.jpg'></div>
                <img src='images/papers/p_2021_TEV_before.jpg'>
              </div>
              <script type="text/javascript">
                function p_2021_TEV_start() {
                  document.getElementById('p_2021_TEV_image').style.opacity = "1";
                }

                function p_2021_TEV_stop() {
                  document.getElementById('p_2021_TEV_image').style.opacity = "0";
                }
                p_2021_TEV_stop()
              </script>
            </td>
            <td style="padding:20px;width:75%;vertical-align:middle">
              <a href="https://pubs.acs.org/doi/10.1021/acssynbio.1c00133">
                <papertitle>Synthetic circuit-driven expression of heterologous enzymes for disease detection</papertitle>
              </a>
              <br>
              Jiang He*,
              Lior Nissim*,
              <strong>Ava P. Soleimany*</strong>,
              Adina Binder-Nissim,
              Heather E. Fleming,
              Timothy K. Lu,
              Sangeeta N. Bhatia
              <br>
              <em>ACS Synthetic Biology</em>, 2021
              <br>
              <a href="https://pubs.acs.org/doi/pdf/10.1021/acssynbio.1c00133">pdf</a> /
              <a href="https://pubs.acs.org/doi/suppl/10.1021/acssynbio.1c00133/suppl_file/sb1c00133_si_001.pdf">supplement</a>
              <br>
              <p> We design a sense-and-respond system that integrates a synthetic
                gene circuit and nanotechnology detection tools for tumor-specific expression
                of heterologous biomarkers.</p>
            </td>
          </tr>

          <tr onmouseout="p_2021_chemprop_stop()" onmouseover="p_2021_chemprop_start()">
            <td style="padding:20px;width:25%;vertical-align:middle">
              <div class="one">
                <div class="two" id='p_2021_chemprop_image'><img src='images/papers/p_2021_chemprop_after.jpg'></div>
                <img src='images/papers/p_2021_chemprop_before.jpg'>
              </div>
              <script type="text/javascript">
                function p_2021_chemprop_start() {
                  document.getElementById('p_2021_chemprop_image').style.opacity = "1";
                }

                function p_2021_chemprop_stop() {
                  document.getElementById('p_2021_chemprop_image').style.opacity = "0";
                }
                p_2021_chemprop_stop()
              </script>
            </td>
            <td style="padding:20px;width:75%;vertical-align:middle">
              <a href="https://pubs.acs.org/doi/10.1021/acscentsci.1c00546">
                <papertitle>Evidential deep learning for guided molecular property prediction and discovery</papertitle>
              </a>
              <br>
              <strong>Ava P. Soleimany*</strong>,
              Alexander Amini*,
              Samuel Goldman*,
              Daniela Rus,
              Sangeeta N. Bhatia,
              Connor W. Coley
              <br>
              <em>ACS Central Science</em>, 2021
              <br>
              <a href="https://pubs.acs.org/doi/pdf/10.1021/acscentsci.1c00546">pdf</a> /
              <a href="https://pubs.acs.org/doi/suppl/10.1021/acscentsci.1c00546/suppl_file/oc1c00546_si_001.pdf">supplement</a>
              <br>
              <p>A fast, scalable approach for uncertainty quantification in neural
                networks enables uncertainty-aware molecular property prediction,
                accelerated property optimization, and guided virtual screening.</p>
            </td>
          </tr>

          <tr onmouseout="p_2020_azp_stop()" onmouseover="p_2020_azp_start()">
            <td style="padding:20px;width:25%;vertical-align:middle">
              <div class="one">
                <div class="two" id='p_2020_azp_image'><img src='images/papers/p_2020_azp_after.jpg'></div>
                <img src='images/papers/p_2020_azp_before.jpg'>
              </div>
              <script type="text/javascript">
                function p_2020_azp_start() {
                  document.getElementById('p_2020_azp_image').style.opacity = "1";
                }

                function p_2020_azp_stop() {
                  document.getElementById('p_2020_azp_image').style.opacity = "0";
                }
                p_2020_azp_stop()
              </script>
            </td>
            <td style="padding:20px;width:75%;vertical-align:middle">
              <a href="https://cancerres.aacrjournals.org/content/81/1/213">
                <papertitle>Activatable zymography probes enable in situ localization of protease dysregulation in cancer</papertitle>
              </a>
              <br>
              <strong>Ava P. Soleimany*</strong>,
              Jesse D. Kirkpatrick*, 
              Susan Su, 
              Jaideep S. Dudani, 
              Qian Zhong, 
              Ahmet Bekdemir,
              Sangeeta N. Bhatia
              <br>
              <em>Cancer Research</em>, 2021
              <br>
              <a href="https://cancerres.aacrjournals.org/content/canres/81/1/213.full.pdf">pdf</a> /
              <a href="https://cancerres.aacrjournals.org/content/canres/suppl/2020/10/24/0008-5472.CAN-20-2410.DC1/247486_2_supp_6653019_qtj5tt.pdf">supplement</a>
              <br>
              <p>We engineer a new class of enzyme activity probes that can be applied to fresh-frozen tissue sections to spatially localize
                protease activty, enabling new insights into the biology of protease dysregulation.</p>
            </td>
          </tr>

          <tr onmouseout="p_2020_evidence_stop()" onmouseover="p_2020_evidence_start()">
            <td style="padding:20px;width:25%;vertical-align:middle">
              <div class="one">
                <div class="two" id='p_2020_evidence_image'><img src='images/papers/p_2020_evidence_before.jpg'></div>
                <img src='images/papers/p_2020_evidence_after.jpg'>
              </div>
              <script type="text/javascript">
                function p_2020_evidence_start() {
                  document.getElementById('p_2020_evidence_image').style.opacity = "1";
                }

                function p_2020_evidence_stop() {
                  document.getElementById('p_2020_evidence_image').style.opacity = "0";
                }
                p_2020_azp_stop()
              </script>
            </td>
            <td style="padding:20px;width:75%;vertical-align:middle">
              <a href="https://papers.nips.cc/paper/2020/hash/aab085461de182608ee9f607f3f7d18f-Abstract.html">
                <papertitle>Deep evidential regression</papertitle>
              </a>
              <br>
              Alexander Amini, 
              Wilko Schwarting,
              <strong>Ava Soleimany</strong>,
              Daniela Rus
              <br>
              <em>NeurIPS</em>, 2020
              <br>
              <a href="https://arxiv.org/pdf/1910.02600.pdf">pdf</a> /
              <a href="https://news.mit.edu/2020/neural-network-uncertainty-1120">press</a>
              <br>
              <p>We develop a novel algorithm for fast, scalable uncertainty quantification in highly complex, non-linear neural networks trained for regression tasks.</p>
            </td>
          </tr>

          <tr onmouseout="p_2020_vax_stop()" onmouseover="p_2020_vax_start()">
            <td style="padding:20px;width:25%;vertical-align:middle">
              <div class="one">
                <div class="two" id='p_2020_vax_image'><img src='images/papers/p_2020_vax_after.jpg'></div>
                <img src='images/papers/p_2020_vax_before.jpg'>
              </div>
              <script type="text/javascript">
                function p_2020_vax_start() {
                  document.getElementById('p_2020_vax_image').style.opacity = "1";
                }

                function p_2020_vax_stop() {
                  document.getElementById('p_2020_vax_image').style.opacity = "0";
                }
                p_2020_vax_stop()
              </script>
            </td>
            <td style="padding:20px;width:75%;vertical-align:middle">
              <a href="https://www.nature.com/articles/s41551-020-0563-4">
                <papertitle>Pharmacokinetic tuning of protein-antigen fusions enhances the immunogenicity of T-cell vaccines</papertitle>
              </a>
              <br>
              Naveen K. Mehta, Roma V. Pradhan, <strong>Ava P. Soleimany</strong>,
              Kelly D. Moynihan, Adrienne M. Rothschilds, Noor Momin, Kavya Rakhra, Jordi Mata-Fink, Sangeeta N. Bhatia, K. Dane Wittrup, Darrell J. Irvine
              <br>
              <em>Nature Biomedical Engineering</em>, 2020
              <br>
              <a href="http://lmrt.mit.edu/sites/de3fault/files/s41551-020-0563-4.pdf">pdf</a> /
              <a href="http://lmrt.mit.edu/sites/default/files/41551_2020_563_MOESM1_ESM.pdf">supplement</a> /
              <a href="https://www.nature.com/articles/s41551-020-0572-3">press</a>
              <br>
              <p>We optimize the immunogenicity of peptide-based antitumor vaccines in mice
                by tuning their pharmacokinetics via fusion of the peptide epitopes to
                protein carriers.</p>
            </td>
          </tr>

          <tr onmouseout="p_2020_abdx_stop()" onmouseover="p_2020_abdx_start()">
            <td style="padding:20px;width:25%;vertical-align:middle">
              <div class="one">
                <div class="two" id='p_2020_abdx_image'><img src='images/papers/p_2020_abdx_after.jpg'></div>
                <img src='images/papers/p_2020_abdx_before.jpg'>
              </div>
              <script type="text/javascript">
                function p_2020_abdx_start() {
                  document.getElementById('p_2020_abdx_image').style.opacity = "1";
                }

                function p_2020_abdx_stop() {
                  document.getElementById('p_2020_abdx_image').style.opacity = "0";
                }
                p_2020_abdx_stop()
              </script>
            </td>
            <td style="padding:20px;width:75%;vertical-align:middle">
              <a href="https://www.cell.com/trends/molecular-medicine/fulltext/S1471-4914(20)30041-1">
                <papertitle>Activity-based diagnostics: an emerging paradigm for disease detection and monitoring</papertitle>
              </a>
              <br>
              <strong>Ava P. Soleimany</strong>,
              Sangeeta N. Bhatia           
              <br>
              <em>Trends in Molecular Medicine</em>, 2020
              <br>
              <a href="http://lmrt.mit.edu/sites/default/files/PIIS1471491420300411.pdf">pdf</a>
              <br>
              <p>Review detailing how integrating techniques from multiple disciplines has developed engineered diagnostics that are selectively
                activated in disease states, highlighting their potential to realize the goals of precision medicine.</p>
            </td>
          </tr>

          <tr onmouseout="p_2020_lung_stop()" onmouseover="p_2020_lung_start()">
            <td style="padding:20px;width:25%;vertical-align:middle">
              <div class="one">
                <div class="two" id='p_2020_lung_image'><img src='images/papers/p_2020_lung_after.jpg'></div>
                <img src='images/papers/p_2020_lung_before.jpg'>
              </div>
              <script type="text/javascript">
                function p_2020_lung_start() {
                  document.getElementById('p_2020_lung_image').style.opacity = "1";
                }

                function p_2020_lung_stop() {
                  document.getElementById('p_2020_lung_image').style.opacity = "0";
                }
                p_2020_lung_stop()
              </script>
            </td>
            <td style="padding:20px;width:75%;vertical-align:middle">
              <a href="https://stm.sciencemag.org/content/12/537/eaaw0262">
                <papertitle>Urinary detection of lung cancer in mice via noninvasive pulmonary protease profiling</papertitle>
              </a>
              <br>
              Jesse D. Kirkpatrick*,
              Andrew D. Warren*, 
              <strong>Ava P. Soleimany*</strong>,
              Peter M. K. Westcott,
              Justin C. Voog,
              Carmen Martin-Alonso, 
              Heather E. Fleming,
              Tuomas Tammela,
              Tyler Jacks,
              Sangeeta N. Bhatia
              <br>
              <em>Science Translational Medicine</em>, 2020
              <br>
              <a href="https://stm.sciencemag.org/content/scitransmed/suppl/2020/03/30/12.537.eaaw0262.DC1/aaw0262_SM.pdf">supplement</a> /
              <a href="http://news.mit.edu/2020/urine-sensor-test-detect-lung-tumors-0401">press</a> /
              <a href="https://www.youtube.com/watch?v=a8C1NWqzRlE">video</a>
              <br>
              <p>We couple protease-responsive nanoparticle sensors with machine learning to engineer a sensitive and specific urinary test for
                lung cancer detection.</p>
            </td>
          </tr>

          <tr onmouseout="p_2020_mri_stop()" onmouseover="p_2020_mri_start()">
            <td style="padding:20px;width:25%;vertical-align:middle">
              <div class="one">
                <div class="two" id='p_2020_mri_image'><img src='images/papers/p_2020_mri_after.jpg'></div>
                <img src='images/papers/p_2020_mri_before.jpg'>
              </div>
              <script type="text/javascript">
                function p_2020_mri_start() {
                  document.getElementById('p_2020_mri_image').style.opacity = "1";
                }

                function p_2020_mri_stop() {
                  document.getElementById('p_2020_mri_image').style.opacity = "0";
                }
                p_2020_mri_stop()
              </script>
            </td>
            <td style="padding:20px;width:75%;vertical-align:middle">
              <a href="https://pubs.acs.org/doi/abs/10.1021/acssynbio.9b00416">
                <papertitle>Genetic encoding of targeted MRI contrast agents for tumor imaging</papertitle>
              </a>
              <br>

              Simone Schuerle,
              Maiko Furubayashi, <strong>Ava P. Soleimany</strong>,
              Tinotenda Gwisai, Wei Huang,
              Christopher A. Voigt,
              Sangeeta N. Bhatia
              <br>
              <em>ACS Synthetic Biology</em>, 2020
              <br>
              <a href="https://pubs.acs.org/doi/suppl/10.1021/acssynbio.9b00416/suppl_file/sb9b00416_si_001.pdf">supplement</a>
              <br>
              <p>Magnetic nanoparticles that display genetically encoded targeting peptides to promote tumor accumulation
              and enhance MRI contrast.</p>
            </td>
          </tr>

          <tr onmouseout="p_2019_gold_stop()" onmouseover="p_2019_gold_start()">
            <td style="padding:20px;width:25%;vertical-align:middle">
              <div class="one">
                <div class="two" id='p_2019_gold_image'><img src='images/papers/p_2019_gold_after.jpg'></div>
                <img src='images/papers/p_2019_gold_before.jpg'>
              </div>
              <script type="text/javascript">
                function p_2019_gold_start() {
                  document.getElementById('p_2019_gold_image').style.opacity = "1";
                }

                function p_2019_gold_stop() {
                  document.getElementById('p_2019_gold_image').style.opacity = "0";
                }
                p_2019_gold_stop()
              </script>
            </td>
            <td style="padding:20px;width:75%;vertical-align:middle">
              <a href="https://www.nature.com/articles/s41565-019-0527-6">
                <papertitle>Renal clearable catalytic gold nanoclusters for in vivo disease monitoring</papertitle>
              </a>
              <br>

              Colleen Loynachan*, <strong>Ava P. Soleimany*</strong>, Jaideep S. Dudani, Yiyang Lin, Adrian Najer, Ahmet Bekdemir, Qu Chen,
              Sangeeta N. Bhatia<sup>&#8224;</sup>,
              Molly M. Stevens<sup>&#8224;</sup>
              <br>
              <em>Nature Nanotechnology</em>, 2019
              <br>
              <a href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41565-019-0527-6/MediaObjects/41565_2019_527_MOESM1_ESM.pdf">supplement</a> /
              <a href="https://spiral.imperial.ac.uk/handle/10044/1/76062">data</a> /
              <a href="https://www.imperial.ac.uk/news/192750/colour-change-urine-test-cancer-shows-potential/">press</a> /
              <a href="https://twitter.com/apsoleimany/status/1168567081853444097">video</a>
              <br>
              <p></p>
              <p>By leveraging the unique properties of catalytic nanomaterials, we develop a simple color-change urine test for detection of cancer in mice.</p>
            </td>
          </tr>


                    <tr onmouseout="p_2019_malaria_stop()" onmouseover="p_2019_malaria_start()">
                      <td style="padding:20px;width:25%;vertical-align:middle">
                        <div class="one">
                          <div class="two" id='p_2019_malaria_image'><img src='images/papers/p_2019_malaria_after.jpg'></div>
                          <img src='images/papers/p_2019_malaria_before.jpg'>
                        </div>
                        <script type="text/javascript">
                          function p_2019_malaria_start() {
                            document.getElementById('p_2019_malaria_image').style.opacity = "1";
                          }

                          function p_2019_malaria_stop() {
                            document.getElementById('p_2019_malaria_image').style.opacity = "0";
                          }
                          p_2019_malaria_stop()
                        </script>
                      </td>
                      <td style="padding:20px;width:75%;vertical-align:middle">
                        <a href="https://arxiv.org/pdf/1912.00262.pdf">
                          <papertitle>Image segmentation of liver stage malaria infection with spatial uncertainty sampling</papertitle>
                        </a>
                        <br>

                        <strong>Ava P. Soleimany</strong>,
                        Harini Suresh,
                        Jose Javier Gonzalez Ortiz,
                        Divya Shanmugam,
                        Nil Gural,
                        John Guttag,
                        Sangeeta N. Bhatia
                        <br>
                        <em>ICML Workshop on Computational Biology</em>, 2019
                        <br>
                        <p>Convolutional neural networks for automated segmentation and uncertainty estimation of microscopy images of
                          malaria infection.</p>
                      </td>
                    </tr>

          <tr onmouseout="p_2019_convection_stop()" onmouseover="p_2019_convection_start()">
            <td style="padding:20px;width:25%;vertical-align:middle">
              <div class="one">
                <img src='images/papers/p_2019_convection_anim.gif'>
              </div>
            </td>
            <td style="padding:20px;width:75%;vertical-align:middle">
              <a href="https://advances.sciencemag.org/content/5/4/eaav4803">
                <papertitle>Synthetic and living micropropellers for convection-enhanced nanoparticle transport</papertitle>
              </a>
              <br>

              <a href="https://hest.ethz.ch/en/department/people/professors/persdetail.MTY3MjM5.TGlzdC85Miw5ODYyMzU0MDI=.html">Simone Schuerle</a>,
              <strong>Ava P. Soleimany</strong>,
              Tiffany Yeh, Giridhar M. Anand, Moritz Haberli, Heather E. Fleming, Nima Mirkhani, Famin Qiu, Sabine Hauert, Xiaopu Wang, Bradley J. Nelson, Sangeeta N. Bhatia
              <br>
              <em>Science Advances</em>, 2019
              <br>
              <a href="https://advances.sciencemag.org/content/advances/suppl/2019/04/19/5.4.eaav4803.DC1/aav4803_SM.pdf"> supplement</a> /
              <a href="http://news.mit.edu/2019/nanoparticles-magnetic-robots-0426"> press</a> /
              <a href="https://www.youtube.com/watch?v=lV4jGbXNtFU">video</a>
              <br>
              <p></p>
              <p>Engineered microrobots that use magnetism to push drug-delivery nanoparticles out of blood vessels and into diseased tissue.</p>
            </td>
          </tr>

          <tr onmouseout="p_2019_debias_stop()" onmouseover="p_2019_debias_start()">
            <td style="padding:20px;width:25%;vertical-align:middle">
              <div class="one">
                <img src='images/papers/p_2019_debias_anim.gif'>
              </div>
            </td>
            <td style="padding:20px;width:75%;vertical-align:middle">
              <a href="https://dl.acm.org/doi/pdf/10.1145/3306618.3314243">
                <papertitle>Uncovering and mitigating algorithmic bias through learned latent structure</papertitle>
              </a>
              <br>
              Alexander Amini*,
              <strong>Ava P. Soleimany*</strong>, Wilko Schwarting,
              Sangeeta N. Bhatia,
              Daniela Rus
              <br>
              <em>AAAI/ACM Conference on Artificial Intelligence, Ethics, and Society</em>, 2019, <em>*Co-first authors</em>
              <br>
              <a href="https://www.csail.mit.edu/news/ai-de-biases-algorithms">MIT press</a> /
              <a href="https://venturebeat.com/2019/01/26/mit-csail-researchers-propose-automated-method-for-debiasing-ai-algorithms/">VentureBeat press</a>
              <br>
              <p></p>
              <p>Generalizable algorithm for mitigating hidden biases within training data,
                by leveraging learned latent distributions to adaptively re-weight the importance of certain data points while training.</p>
            </td>
          </tr>

          <tr onmouseout="p_2017_spatial_stop()" onmouseover="p_2017_spatial_start()">
            <td style="padding:20px;width:25%;vertical-align:middle">
              <div class="one">
                <div class="two" id='p_2017_spatial_image'><img src='images/papers/p_2017_spatial_after.jpg'></div>
                <img src='images/papers/p_2017_spatial_before.jpg'>
              </div>
              <script type="text/javascript">
                function p_2017_spatial_start() {
                  document.getElementById('p_2017_spatial_image').style.opacity = "1";
                }

                function p_2017_spatial_stop() {
                  document.getElementById('p_2017_spatial_image').style.opacity = "0";
                }
                p_2017_spatial_stop()
              </script>
            </td>
            <td style="padding:20px;width:75%;vertical-align:middle">
              <a href="https://arxiv.org/pdf/1805.04829.pdf">
                <papertitle>Spatial uncertainty sampling for end-to-end control</papertitle>
              </a>
              <br>
              Alexander Amini,
              <strong>Ava Soleimany</strong>,
              Sertac Karaman,
              Daniela Rus,
              <br>
              <em>NeurIPS Workshop on Bayesian Deep Learning</em>, 2017
              <br>
              <p>Estimating uncertainty in neural networks for end-to-end control by exploiting feature map correlations during training.</p>
            </td>
          </tr>

          <tr onmouseout="p_2016_science_stop()" onmouseover="p_2016_science_start()">
            <td style="padding:20px;width:25%;vertical-align:middle">
              <div class="one">
                <div class="two" id='p_2016_science_image'><img src='images/papers/p_2016_science_after.jpg'></div>
                <img src='images/papers/p_2016_science_before.jpg'>
              </div>
              <script type="text/javascript">
                function p_2016_science_start() {
                  document.getElementById('p_2016_science_image').style.opacity = "1";
                }

                function p_2016_science_stop() {
                  document.getElementById('p_2016_science_image').style.opacity = "0";
                }
                p_2016_science_stop()
              </script>
            </td>
            <td style="padding:20px;width:75%;vertical-align:middle">
              <a href="https://science.sciencemag.org/content/353/6297/aad8559">
                <papertitle>Synthetic recombinase-based state machines in living cells</papertitle>
              </a>
              <br>
              Nathaniel Roquet,
              <strong>Ava P. Soleimany</strong>,
              Alyssa C. Ferris,
              Scott Aaronson,
              Timothy K. Lu
              <br>
              <em>Science</em>, 2016
              <br>
              <a href="https://science.sciencemag.org/content/sci/suppl/2016/07/20/353.6297.aad8559.DC1/Roquet.SM.pdf">supplement</a> /
              <a href="http://news.mit.edu/2016/biological-circuit-cells-remember-respond-stimuli-0721">press</a> /
              <a href="https://www.scottaaronson.com/blog/?p=2862">blog</a>
              <p></p>
              <p>Programming biological state machines that enable cells to remember and respond to a series of events.</p>
            </td>
          </tr>

        </tbody></table>

        <table width="100%" align="center" border="0" cellspacing="0" cellpadding="20"><tbody>
          <tr>
            <td>
              <heading id="teaching">Teaching and Leadership</heading>
              <p>
              In addition to research, I am passionate about education and leadership, and strive to help and empower others to excel in their own pursuits.
              </p>
            </td>
          </tr>
        </tbody></table>

        <table width="100%" align="center" border="0" cellpadding="20"><tbody>
          <tr onmouseout="p_deep_learning_stop()" onmouseover="p_deep_learning_start()">
            <td style="padding:20px;width:25%;vertical-align:middle">
              <div class="one">
                <div class="two" id='p_deep_learning_image'><img src='images/p_deep_learning_after.png'></div>
                <img src='images/p_deep_learning_before.png'>
              </div>
              <script type="text/javascript">
                function p_deep_learning_start() {
                  document.getElementById('p_deep_learning_image').style.opacity = "1";
                }

                function p_deep_learning_stop() {
                  document.getElementById('p_deep_learning_image').style.opacity = "0";
                }
                p_2016_science_stop()
              </script>
            </td>
            <td style="padding:20px;width:75%;vertical-align:middle">
              <a href="http://introtodeeplearning.com">
                <papertitle>Introduction to Deep Learning, MIT 6.S191</papertitle>
              </a>
              <p>I am an organizer and lecturer for <a href="http://introtodeeplearning.com">Introduction to Deep Learning (6.S191)</a>, MITâ€™s official introductory course on deep learning foundations and applications.
              Together with <a href="https://www.mit.edu/~amini/">Alexander Amini</a>, I have organized and developed all aspects of the course, including developing the curriculum, teaching the lectures, creating software labs, and collaborating with industry sponsors.
              All materials can be found online on the <a href="http://introtodeeplearning.com">course website</a>.</p>
            </td>
          </tr>

      <table width="100%" align="center" border="0" cellpadding="20"><tbody>
        <tr>
          <td style="padding:20px;width:25%;vertical-align:middle"><img src="images/momentum.jpg"></td>
          <td width="75%" valign="center">
            <a href="https://momentumai.org/">Co-founder, Momentum AI</a>
            <p>I am a co-founder and director for Momentum AI, an outreach program that teaches AI and machine learning to under-resourced and under-served high school
              students from the greater Boston area. Our two-week capston program is a free, project-based deep dive into AI and is held on MIT's campus.
            <br>
          </td>
        </tr>

        <tr>
          <td style="padding:20px;width:25%;vertical-align:middle"><img src="images/harvard.png"></td>
          <td width="75%" valign="center">
            <a href="https://locator.tlt.harvard.edu/course/colgsas-122422">Teaching Fellow, Harvard MCB294, Fall 2019</a>, with <a href="https://www.mcb.harvard.edu/directory/nancy-kleckner/">Nancy Kleckner</a>
            <br>
          </td>
        </tr>

        <tr>
          <td style="padding:20px;width:25%;vertical-align:middle">
            <img src="images/mit.png" alt="mit_teaching">
          </td>
          <td width="75%" valign="center">
            <a href="https://stellar.mit.edu/S/course/7/sp16/7.05/index.html"> Teaching Assistant, MIT 7.05, Spring 2016</a>, with <a href="https://ki.mit.edu/people/faculty/vanderheiden">Matt Vander Heiden</a> and <a href="https://ki.mit.edu/people/faculty/yaffe">Mike Yaffe</a>
            <br>
            <br>
            <a href="https://stellar.mit.edu/S/course/7/sp15/7.05/index.html"> Teaching Assistant, MIT 7.05, Spring 2015</a>, with <a href="https://ki.mit.edu/people/faculty/vanderheiden">Matt Vander Heiden</a> and <a href="https://ki.mit.edu/people/faculty/yaffe">Mike Yaffe</a>
          </td>
        </tr>
      </tbody></table>



      <tr>
        <td style="padding:20px;width:100%;vertical-align:middle">
          <heading id="talks">Select Publicly-Available Talks</heading>
          <ul>
            <li>[Feb. 2025] <strong><a href="https://youtu.be/5K0YemTKu-w?si=i6WOJI8-TAsUz6cm">AI for Precision Health: Learning the language of nature and patients</a></strong>, Microsoft Research Forum.</li>
            <li>[Sep. 2024] <strong><a href="https://youtu.be/7AMRTLlbl9E?si=cC3h85zCikenPS0m">Bridging biophysics and AI for generative protein design</a></strong>, MLCB Keynote.</li>
            <li>[Apr. 2024] <strong><a href="https://youtu.be/0KTlM4Oav8E?si=joKM6FvKQpY5-lQM">Learning the language of biology: towards precision medicine</a></strong>, MIT IIA Summit.</li>
            <li>[Jan. 2024] <strong><a href="https://youtu.be/jA7xwOsFffU?si=jKVHM0JEx5PBCtyY">How will AI transform precision medicine?</a></strong>, World Economic Forum.</li>
            <li>[Nov. 2023] <strong><a href="https://youtu.be/L_PcZaXjip0?si=WzmVTqKoyO4iQ3H7">Protein design with generative diffusion models</a></strong>, Broad Institute MIA Seminar.</li>
            <li>[Oct. 2022] <strong><a href="https://youtu.be/lnkZjdtHoiY?si=JA44qhf3KeyCqY8H&t=459">Optimizing ex vivo organoids for precision cancer medicine</a></strong>, Microsoft Research Forum.</li>
            <li>[Oct. 2021] <strong><a href="https://youtu.be/0yS8psNSkCo?si=InsWhV5XyVSh7LTs">Leveraging uncertainty in machine learning to bridge computation and experimentation</a></strong>, Microsoft Research Forum.</li>
            <!-- <a href="#"> &#8681 load more</a> -->


            
          </ul>
        </td>
      </tr>


      </td>
    </tr>
  </table>

  <p>
    <a href="https://jonbarron.info/">page template</a>
  </p>
</body>

</html>
